tests/correctness/vadd_s8_rvv_test/vadd_s8_rvv_test.c

```diff
@@ -0,0 +1,74 @@
+// Copyright 2024 10xEngineers
+
+#include "../../../common/print.h"
+
+#ifdef ARM_NEON
+    #include <arm_neon.h>
+#else
+#ifdef RISCV_VECTOR
+    #include "../../../common/rhal_base.h"
+#endif
```

Comment:
it should follow an if elif and else #error scheme not two if defs and endifs.

Reply:
done